pkgs/applications/misc/pop-launcher/default.nix

```diff
@@ -0,0 +1,46 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, just
+, runtimeShell
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pop-launcher";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "launcher";
+    rev = "${version}";
+    sha256 = "sha256-BQAO9IodZxGgV8iBmUaOF0yDbAMVDFslKCqlh3pBnb0=";
+  };
+
+  postPatch = ''
+    substituteInPlace justfile \
+        --replace '#!/usr/bin/env sh' '#!${runtimeShell}'
+    substituteInPlace src/lib.rs \
+        --replace '/usr/lib/pop-launcher' "$out/share/pop-launcher"
+    substituteInPlace plugins/src/scripts/mod.rs \
+        --replace '/usr/lib/pop-launcher' "$out/share/pop-launcher"
+  '';
+
+  cargoSha256 = "sha256-cTvrq0fH057UIx/O9u8zHMsg+psMGg1q9klV5OMxtok=";
+
+  nativeBuildInputs = [ just ];
+
+  buildPhase = ''
+    just
```

Comment:
Does just support -j (jobs) or a similar flag like make does?

Reply:
It doesn't appear to. But it also isn't really meant as a replacement for a build tool, since it's more of a command runner.

Reply:
I think that is just invoking cargo commands, right? then we should use the nixpkgs tooling for cargo rather than just.

Reply:
@SuperSandro2000 it does a bit more than that, and who knows what it might do in the future. However I do agree that as it is now, replacing it for just the nixpkgs handling of cargo with some additional postBuildPhase script might be the ideal and more straightforward way of doing this yes.

Reply:
looks better now